### DISCUSSION TITLE
My account got suspended suddenly without any previous warnings

### DISCUSSION THREAD

**Discussion by engtechno** [community/community#146935] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

@github-staff

Today I didn't even touch my account @ammarbasuony or do any operation. And suddenly received two emails in the same minitue. The account is flagged and then suspended

I'm just using my account to store my code, collaborate with the codebase of the company I'm working for, and do simple actions for deploying my code to my server.

Can I have any help as all my work and livelihood is affected because of that

**Comment by techy4shri** [comment]
Raise a ticket with support using the appeal option which comes when you cannot sign in. Somehow this has happened to many. Be sure to check with them once in a while for speeding up the process.
Mark this as answer if it helped :)
